### ISSUE TITLE
Fix deprecation warnings

### ISSUE THREAD

**Issue by varac** [systemli/ansible-role-letsencrypt#2]
I get this warning using the letsencrypt role with ansible 2.4:

```
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions. This feature will be removed in a future release. 
Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

I guess it's these includes in main.yml:

```
tasks/main.yml:- include: install.yml
tasks/main.yml:- include: certificate.yml
```

**Comment by 0x46616c6b** [comment]
We support Ansible 2.2, so we can't fix this warnings at the moment without dropping support for the version 2.2. One reason is, that v2.2 is the latest version in Debian Jessie.

**Comment by t2d** [comment]
Fixed by 6bbe07af564aa6d61556a9078cbd09876511dcaf
